src/popup/pages/Assets/AssetDetailsInfo.vue

```diff
@@ -199,19 +215,23 @@ export default defineComponent({
     Tokens,
     IonPage,
     IonContent,
+    LinkButton,
   },
 
   setup() {
     const { sharedAssetDetails } = useAssetDetails();
     const { formatCurrency } = useCurrencies();
 
-    const assetData = computed((): IAsset => sharedAssetDetails.tokenData || {});
+    const assetData = computed(() => sharedAssetDetails.tokenData || {});
+    const adapter = computed(() => ProtocolAdapterFactory.getAdapter(assetData.value?.protocol));
     const assetContractId = computed(() => assetData.value.contractId);
     const tokens = computed(() => sharedAssetDetails.tokens);
     const tokenPairs = computed(() => sharedAssetDetails.tokenPairs || {});
     const tokenBalance = computed(() => sharedAssetDetails.tokenBalance);
     const isAssetCoin = computed(() => isCoin(assetContractId.value));
     const decimals = computed(() => assetData.value?.decimals || tokenBalance.value?.decimals);
+    const assetCoinGeckoLink = computed(() => `https://www.coingecko.com/en/coins/${adapter.value.getCoinGeckoCoinId()}`);
+    const displayCoinGeckoLink = computed(() => assetCoinGeckoLink.value.replace('https://', '').replace('en/coins', '...'));
```

Comment:
I suggest to rename those 2 constants to: `coinGeckoLinkUrl` and `coinGeckoLinkLabel` to make it more readable.